test/common/stats/thread_local_store_test.cc

```diff
@@ -775,6 +775,79 @@ TEST_F(StatsThreadLocalStoreTest, SharedScopes) {
   tls_.shutdownThread();
 }
 
+TEST_F(StatsThreadLocalStoreTest, ExtractAndAppendTagsFixedValue) {
```

Comment:
can you also make a test where you are using the regex-extracted tags?

To do this you'll see to calll
```
  store_->setTagProducer(std::make_unique<Stats::TagProducerImpl>(stats_config));
```
if that's not already happened. and then make a stat like "cluster.cluster_name.leaf"

Reply:
The two tests below this one are regex testing, unless I misunderstood what you mean?

Reply:
Sorry, I meant using the builtin expressions (not necessarily regexes actually) to automatically extract things like cluster.foo.bar into tag-extracted name "bar" with tag name="cluster" value="bar", based on the token and regex patterns around https://github.com/envoyproxy/envoy/blob/f88e29647931c7c1f1d2ff181e4d95602b0928fa/source/common/config/well_known_names.cc#L128

Reply:
Got it, added.